docs/dev/swagger-source.json

```diff
@@ -1287,6 +1291,336 @@
+					},
+					{
+						"description": "The details about the contributor that you want to edit. All params are optional. When a param is omitted, the existing value of that param will not be modified.",
+						"in": "body",
+						"schema": {
+							"$ref": "#/definitions/ContributorEditable"
+						}
+					}
+				],
+				"responses": {
+					"200": {
+						"x-summary": "Success",
+						"description": "The contributor that was just edited with the new values after the update has occurred.",
+						"schema": {
+							"$ref": "#/definitions/Contributor"
+						}
+					},
+					"400": {
+						"x-summary": "Request Invalid",
+						"description": "The values you sent with the request did not pass validation. The response will contain information about which parameters failed validation.",
+						"schema": {
+							"$ref": "#/definitions/InvalidParametersResponse"
+						}
+					},
+					"403": {
+						"x-summary": "Request Unauthorized",
+						"description": "This error indicates one of the following:\n* The publication you want to edit does not belong to this submission.\n* This publication is published and can not be edited. You must create and publish a new version.\n* You do not have permission to edit this publication.\n* You are not allowed to modify the `status` of a publication. You must use the `/submissions/{submissionId}/publications/{publicationId}/publish` and `.../unpublish` endpoints to modify the status of a publication.",
+						"schema": {
+							"$ref": "#/definitions/ErrorResponse"
+						}
+					},
+					"404": {
+						"x-summary": "Request Not Found",
+						"description": "The publication could not be found.",
+						"schema": {
+							"$ref": "#/definitions/ErrorResponse"
+						}
+					}
+				}
+			},
+			"delete": {
+				"tags": [
+					"Submissions / Publications / Contributors"
+				],
+				"summary": "Delete a contributor.",
+				"description": "Only journal managers and subeditors can make a request to this endpoint.",
+				"parameters": [
+					{
+						"name": "submissionId",
+						"in": "path",
+						"description": "Submission ID",
+						"required": true,
+						"type": "integer"
+					},
+					{
+						"name": "publicationId",
+						"in": "path",
+						"description": "Publication ID",
+						"required": true,
+						"type": "integer"
+					},
+					{
+						"name": "contributorId",
+						"in": "path",
+						"description": "Contributor ID",
+						"required": true,
+						"type": "integer"
+					}
+				],
+				"responses": {
+					"200": {
+						"x-summary": "Success",
+						"description": "The publication that was just deleted.",
+						"schema": {
+							"$ref": "#/definitions/Publication"
+						}
+					},
+					"403": {
+						"x-summary": "Request Unauthorized",
+						"description": "This may mean one of the following:\n* You are not allowed to delete this publication.\n* The publication does not belong to this submission.\n* This publication is published and can not be deleted.",
```

Comment:
> This may mean one of the following:
>
> * You are not allowed to edit the contributors for this publication.
> * This contributor is assigned to a publication that is not part of the requested submission.
> * This contributor is assigned to a published publication and can not be edited.

Reply:
Changed